### ISSUE TITLE
Feature request / potential far-future addition: Mooooar cursive

### ISSUE THREAD

**Issue by arrowtype** [arrowtype/recursive#313]
TL;DR: a more-cursive `f`, `l`, `r`, `i`, and `s` _cooould_ be added, but I believe the current "true italic" versions of these work better in the overall design. But, maybe in the future, I or someone else could work to add these – but it might require a bigger adjustment that just adding a few alternates.

<sub><sup>Of course, even without adjusting the rest of the Italics to make more-cursive forms work well, each additional glyph in this family is (much) more work than you might expect. So, this is partly an issue of timing and availability.</sup></sub>
<sub><sup>See the following explanation for details: https://github.com/arrowtype/recursive/issues/218#issuecomment-551195560</sup></sub>

## The basic idea

In https://github.com/arrowtype/recursive/issues/153, @giovanicascaes gave the following request:

> Wondering how can I give some feedback to you guys. Maybe the only thing I can tell you right now is that a semicursive italic variant, like Operator Mono or Dank Mono one's, would be very appreciated, since these are very popular typefaces, much of it because of that feature. Of course, it would be a lot of work, tough.

My response was:

> Thanks! In my view, this is already the case – almost all lowercase letters have specifically semi-cursive variants in the Italic styles. A single-story /a, long /f, simplified /i, /l, and /r, outstrokes on /n, /h, /s, etc. Of course, the italics don’t look cursive to the same degree because it doesn’t include things like the looped /l and /f and the scripty /r and /a. This is for two main reasons:
> 
> I don’t want to make something that already exists. Operator is a great design, and I’m not trying to compete with it or make a free alternative, but rather to make something original and unique.
To me, the “cursive” forms don’t seem as immediately readable and they don’t work that well within this design, aesthetically (especially due to the Linear–Casual axis). I’ve tried drawing them, and I just don’t think it looks that good.
>
> Still, I’m not 100% against the idea, and it could potentially be the sort of thing added into a stylistic set or even a forked version. But, yes, it also becomes a factor of time: in the near term, I need to finish the project and do other work. In the long term, I may change my mind and add them in.

@davelab6 echoed Giovanni's curiosity with:

> I'm curious what these sketches look like :) No worries if they were not good enough to share with anyone outside your house though XD

So, this is a quick issue to record a few of my thoughts and experiments around this. 

## What I've tried so far

Current Mono Casual ExtraBold Italics:

![image](https://user-images.githubusercontent.com/45946693/74194096-94f87300-4c26-11ea-8f7e-4d58508a3755.png)

Current Mono Linear ExtraBold Italics:

![image](https://user-images.githubusercontent.com/45946693/74194691-c3c31900-4c27-11ea-9057-549222253335.png)

Right now, the italics have the following goals (in addition to being monospace and fitting in with the overall aesthetics of the typeface):

1. Have nice-looking spacing within the confines of a fixed width
2. Be instantly and easily recognizable
3. Look good in both Casual and Linear subfamilies, without changing overall construction (strokes should change, but overall shape should stay consistent to fascilitate animated transitions)

On goal 1, "cursive" construction actually has a slight edge, because the in and out-strokes can fill in spacing near the baseline, whereas traditional mono italic `l` and `i` forms can leave big gaps. 

On goal 2, cursive letters are at a disadvantage – their shaping is just so different from what most people read most of the time. Especially for folks who didn't have to learn to write cursive in grade school, these forms are weird and take getting-used-to.

On goal 3, there are the biggest visual problems. 

- In the Casual, Italics have a "wavy" form borrowed from brush-pen writing. The simple italic `i` and `l` echo this, and so help reinforce the overall texture and rhythm of the typeface. If cursive forms were introduced, it maaay mean reconsidering this wavy approach. Basically, I worry that it might mean redrawing the entire thing almost from scratch. 😅
- In the Linear, the simple italics echo the overall stripped-back utilitarianism of the other letters. Very early on in the project (back in early 2018), I worked on things in this area, but they often felt a little too much like 1950s car or appliance lettering, which didn't feel right.

Early exploration of connected cursive (early March 2018):

![image](https://user-images.githubusercontent.com/45946693/74196621-a1cb9580-4c2b-11ea-8e66-d8d64cccc33f.png)

Early exploration of disconnected cursive (late March 2018):

![image](https://user-images.githubusercontent.com/45946693/74196617-a001d200-4c2b-11ea-8b82-18033885690c.png)

More recently (within the last few weeks), I dipped my toes into the cursive waters again, and these experiments are in the Mono Casual B Slanted master ([permalink to latest commit of this](https://github.com/arrowtype/recursive/tree/d0a07e5db1bd758170646b8ab107e941889b3104/src/masters/mono/Recursive%20Mono-Casual%20B%20Slanted.ufo)). This looks okaaay, but kind of messy. To really understand it, however, these glyphs would need to be drawn in at least 3 other masters and put into test files for use in code & design mockups. But, I'm pretty unconvinced with what I have so far and especially unconvinced that they would work particularly well in Linear styles.

![image](https://user-images.githubusercontent.com/45946693/74194003-6da1a600-4c26-11ea-8a66-a5ed566eb6b5.png)

<details>
<summary><b>(here's the test string I used in Space Center)</b> (Click to expand)</summary>

```
Re/l.cursive aps/i.cursive ng F/l.cursive a/r.cursive e-Ups Cova/r.cursive y Yawny Footpace Bèa/r.cursive na/i.cursive ses Mohe/l.cursive  B/r.cursive ead/f.cursive /r.cursive u/i.cursive ts Ins/i.cursive d/i.cursive ousness S/i.cursive sse/r.cursive a/r.cursive /i.cursive es Looke/r.cursive s-On Subsumed Neph/r.cursive /i.cursive c I/l.cursive /l.cursive at/i.cursive on Gou/r.cursive mand Jacob/i.cursive n/i.cursive ze Decads R/i.cursive ght-Ang/l.cursive e Chabouks Ru/r.cursive a/l.cursive /i.cursive st Loheng/r.cursive /i.cursive n Sympath/i.cursive ses F/i.cursive na/l.cursive /i.cursive se Pantoscope Stone-B/r.cursive u/i.cursive se Ove/r.cursive washes Te/l.cursive eg/r.cursive aph-Cab/l.cursive e Meda/l.cursive /l.cursive /i.cursive on Snedded Matched G/l.cursive /i.cursive tte/r.cursive ed Monoam/i.cursive nes U/r.cursive ns Unde/r.cursive b/i.cursive dd/i.cursive ng Lo/i.cursive te/r.cursive /i.cursive ng Ha/i.cursive duck Ove/r.cursive stud/i.cursive es O/r.cursive n/i.cursive thomo/r.cursive ph Rh/i.cursive zopus Ensna/r.cursive /l.cursive  B/r.cursive ehons D/r.cursive /i.cursive nkab/l.cursive e Johann/i.cursive sbe/r.cursive ge/r.cursive  Up/r.cursive ate Fa/r.cursive th/i.cursive ng Bet/r.cursive oth/i.cursive ng Tute/l.cursive a/r.cursive  Revo/l.cursive ved Exmoo/r.cursive  De/f.cursive /i.cursive/r.cursive s/i.cursive ve H/i.cursive b/r.cursive /i.cursive nat/i.cursive on
```

</details>

It's possible that this would be a good idea for something the enhance the project with, at some point in the future, along with other areas that could be useful:
- True-italic caps (e.g. alternate, more-scripty forms of E, C, S, G, N, T, etc)
- Cyrillic or other language extensions
- Other styles/alternates (e.g. casual blackletter)

**Comment by giovanicascaes** [comment]
Your sketches look great :)
You have strong arguments to not touching the cursive topic for now. I really understand your point.

**Comment by arrowtype** [comment]
Thanks, @giovanicascaes! Hope my explanation wasn't overly intense; I'm also just trying to clarify my own thoughts on this by writing, and leave a record for others that will no doubt come along with the same question, later. It's not so much that I don't want to or think it's a bad idea outright, but more that it would be a pretty big additional lift to get it right. :)

**Comment by arrowtype** [comment]
Closing this issue to keep the issues clean but keep it around for others who might search for it.
